src/panels/lovelace/editor/config-elements/hui-alarm-panel-card-editor.ts

```diff
@@ -71,7 +71,7 @@ export class HuiAlarmPanelCardEditor extends hassLocalizeLitMixin(LitElement)
             .hass="${this.hass}"
             .value="${this._entity}"
             .configValue=${"entity"}
-            .domainFilter=${"alarm_control_panel"}
+            domain-filter=${"alarm_control_panel"}
```

Comment:
Since it's not dynamic, don't even wrap it in `${…}`. Just make it `domain-filter="alarm_control_panel"`

Reply:
On a side note, Polymer automatically translated kebab-case attributes to camelCase properties, Lit does not. So if we ever change `<ha-entity-picker>` to Lit, we should update every location that we use `domain-filter`. We might want to add a comment to `<ha-entity-picker>` so it's not forgotten.